### ISSUE TITLE
Retry button - for finding Dropbox folders located on external drives

### ISSUE THREAD

**Issue by mstgrv** [samschott/maestral#846]
**Is your feature request related to a problem? Please describe.**
I have my Dropbox folder on an external drive. When I reboot, Maestral always launches before the drive has had a chance to mount, so I get a popup that Maestral can't find my Dropbox folder every time I restart, which is annoying when the only issue is the small time gap between Maestral launching and the drive mounting successfully.

**Describe the solution you'd like**
It would be great to add a "Retry" button to the dialogue box that pops up, so that Maestral can simply try to find the existing folder again when it has had a chance to mount,

**Describe alternatives you've considered**
None of the existing options in the dialogue box seem appropriate for this - I don't want to move the folder, or do anything that will trigger an unnecessary rescan or resync. I just want Maestral to retry the existing folder location, once I know that drive has mounted and is good to go.

**Comment by samschott** [comment]
> None of the existing options in the dialogue box seem appropriate for this

I would argue that the "Quit" option is appropriate, followed by starting Maestral again once the drive was mounted. That being said, a retry button certainly would be an option.

The most elegant solution would be to create you own startup entry (using launchd on macOS, systemd or an XDG desktop entry on Linux) which either waits explicitly until the drive is mounted, or which waits a constant amount of time after login before starting Maestral. This would remove the need to press any button at all.

**Comment by mstgrv** [comment]
Agreed a customised startup with launchd would be great, but it's a bit of an ask for most people (including me) to learn OS tools like that, when a Retry button feels just as elegant in this situation. Quit & restart is the least elegant option, but it's the one I'm using for the moment.

**Comment by RokeJulianLockhart** [comment]
**https://github.com/samschott/maestral/issues/846#issuecomment-1450915473**

> or which waits a constant amount of time after login before starting Maestral. This would remove the need to press any button at all.

@samschott, time-based solutions simply don't work on slow hardware. It would need to verify that all drives have been mounted (of course, in addition to the retry button since unmounture at any time remains possible).
